src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlConstants.cs

```diff
@@ -16,6 +16,8 @@ internal static class XamlConstants
 		public const string BaseXamlNamespace = RootUINamespace + ".Xaml";
 		public const string UnoXamlNamespace = "Windows.UI.Xaml";
 
+		public const char XBindSubstitute = '↔';
```

Comment:
What is that??

Reply:
See:
https://github.com/unoplatform/uno/pull/2429/files#diff-8791012553acc33c896cb7e78896157bR134